test/react-native/features/app.feature

```diff
@@ -30,63 +34,84 @@ Scenario: Unhandled JS error
   And the exception "message" equals "AppJsUnhandledScenario"
   And the event "unhandled" is true
   And the event "app.version" equals "1.2.3"
-  And the event "app.versionCode" equals 1
+  # Parameter not present on iOS devices
```

Comment:
I do see this for iOS events on the dashboard, though it may only be included in a release build?

![image](https://user-images.githubusercontent.com/609579/87793339-06bf9300-c83d-11ea-8a36-cc87d6c88b8c.png)

Reply:
I understood from [here](https://github.com/bugsnag/platforms-notifier-spec/blob/master/app.md#versioncode-1) that it was Android specific.  @fractalwrench do you know if that's the case?

Reply:
The versionCode does appear to be populated on iOS events so the iOS implementation and spec are inconsistent.